source/reference/mongosync.txt

```diff
@@ -1,9 +1,95 @@
 .. _c2c-mongosync:
 
+.. default-domain:: mongodb
+
+.. program:: mongosync
+
 =============
 ``mongosync``
 =============
 
-.. default-domain:: mongodb
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Synopsis
+--------
+
+The ``mongosync`` binary is the primary process used in Cluster to Cluster Sync.
+It migrates data from one cluster to another and can keep the clusters
+in continuous sync.
+
+This document provides a complete overview of all command-line options.
+
+Options
+-------
+
+Global Options
+~~~~~~~~~~~~~~
+
+.. option:: --cluster0 <URI>
+
+   Sets the URI of the first cluster.  The first cluster can serve as either the
+   source cluster or the destination cluster in the synchronization process.
+
+
+.. option:: --cluster1 <URI>
+
+   Sets the URI of the second cluster.  The second cluster can serve as either
+   the source cluster or the destination cluster in the synchronization process.
+
+
+.. option:: --verbosity <level>
+
+   *Default*: ``INFO``
+
+   Sets the verbosity level to use in log messages.  Cluster to Cluster Sync logs
+   all messages at the specified level as well as any messages at less severe levels.
+
+   The ``--verbosity`` option supports the following values:
+
+   - ``TRACE``
```

Comment:
Is there any documentation on what is logged at each level? 

Consider adding "Logging" as a subsection of Behavior.

Reply:
@davemungo, We don't have any information on this at the moment, (I was planning on a table with descriptions, but couldn't find any information to fill out the descriptions).  

It would be good to have something brief here and then a top-level or second-level page on Logging to cover more detail, but it may work better timewise to add that incrementally after World.

Reply:
Understood. 

Would you open a ticket in Jira to add the log level details?

Reply:
Done